scripts/compile_coq.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/sh -e
```

Comment:
Have you checked that the script indeed works with `sh` and not just `bash` ? I have, in the past, wrote scripts with a `sh` shebang, but which only worked for `bash` (because on my setup `sh` is symlinked to `bash`), and it sometimes can break on other peoples's system.

Reply:
I replaced /bin/sh with /bin/bash -- on my system sh was indeed symlinked to bash.